__tests__/app.test.js

```diff
@@ -22,4 +22,37 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics }  = body;
+        expect(Array.isArray(topics)).toBe(true);
```

Comment:
Instead of this, please get into the habit of checking the length of the array. As you are using a forEach on the next line, checking it's an array is a bit redundant. If an empty array was returned, your assertions in your forEach would be skipped over which is why it's important that we check the length before using the array method. Other than this, great test.